src/program.js

```diff
@@ -145,7 +155,19 @@ export class Program {
         });
       }
 
-      await runCommand(argv, {shouldExitProgram});
+      let argvFromConfig = { ...argv };
+      if (argv.config) {
+        const configFileName = path.resolve(argv.config);
+        const configObject = loadJSConfigFile(configFileName);
+        argvFromConfig = applyConfigToArgv({
```

Comment:
Thanks for correcting this. It also needs test coverage. When I comment out your fix, the tests all pass -- at least one should fail.